internal/cache/file/downloader/job_test.go

```diff
@@ -60,7 +60,7 @@ func (dt *downloaderTest) initJobTest(objectName string, objectContent []byte, s
 	err := storageutil.CreateObjects(ctx, dt.bucket, objects)
 	AssertEq(nil, err)
 	dt.object = dt.getMinObject(objectName)
-	dt.fileSpec = data.FileSpec{Path: dt.fileCachePath(dt.bucket.Name(), dt.object.Name), Perm: util.DefaultFilePerm}
+	dt.fileSpec = data.FileSpec{Path: dt.fileCachePath(dt.bucket.Name(), dt.object.Name), FilePerm: util.DefaultFilePerm}
```

Comment:
add DirPerm to data.FileSpec. I think that's the cause of failure in presubmit tests. (it is taking default as 000 probably)

Reply:
hmm could be...added default dirPerm it should fail on my local, maybe it takes system's default permission, server machine might have stricter permissions than cloudtop machines.